src/inputSources/mouseInput.js

```diff
@@ -108,15 +110,10 @@ function mouseDown (e) {
     type: eventType
   };
 
-  const event = external.$.Event(eventType, eventData);
-
-  external.$(eventData.element).trigger(event, eventData);
+  triggerEvent(eventData.element, eventType, eventData);
 
-  if (event.isImmediatePropagationStopped() === false) {
```

Comment:
Any specific reason this was removed? The rest looks fine to me.

Reply:
I made a mistake now I see! I thought that events triggered asynchronously, but it seems they trigger synchronously. DispatchEvent returns a boolean that indicates if any listeners called `preventDefault`.

I guess you learn something everyday!

I'll make a change to reinstate the old behaviour.

Reply:
I have fixed it and some other minor issues, the old behaviour should be back now.